Cargo.toml

```diff
@@ -1,11 +1,42 @@
+[workspace]
+resolver = "2"
+members = [
+    "fish-rust/widestring-suffix",
+]
+
+[workspace.package]
+rust-version = "1.67"
+edition = "2021"
+
+# TODO: Move fish-rust to src, make it the root package of this workspace
+
+[patch.crates-io]
+cc = { git = "https://github.com/mqudsi/cc-rs", branch = "fish" }
+cxx = { git = "https://github.com/fish-shell/cxx", branch = "fish" }
+cxx-gen = { git = "https://github.com/fish-shell/cxx", branch = "fish" }
+autocxx = { git = "https://github.com/fish-shell/autocxx", branch = "fish" }
+autocxx-build = { git = "https://github.com/fish-shell/autocxx", branch = "fish" }
+autocxx-bindgen = { git = "https://github.com/fish-shell/autocxx-bindgen", branch = "fish" }
+
+[patch.'https://github.com/fish-shell/cxx']
+cc = { git = "https://github.com/mqudsi/cc-rs", branch = "fish" }
+
+[patch.'https://github.com/fish-shell/autocxx']
+cc = { git = "https://github.com/mqudsi/cc-rs", branch = "fish" }
+
+
+[profile.release]
+overflow-checks = true
+
 [package]
 name = "fish-rust"
 version = "0.1.0"
-edition = "2021"
```

Comment:
what's the selling point? I like repo-relative paths but that's not really an important consideration

Reply:
I look at individual commits so I totally didn't mind this one